promql/promqltest/testdata/subquery.test

```diff
@@ -49,16 +49,16 @@ load 10s
   metric3 0+3x1000
 
 eval instant at 1000s sum_over_time(metric1[30s:10s])
-  {} 394
+  {} 297
 
 # This is (394*2 - 100), because other than the last 100 at 1000s,
 # everything else is repeated with the 5s step.
 eval instant at 1000s sum_over_time(metric1[30s:5s])
-  {} 688
+  {} 591
 
 # Offset is aligned with the step.
 eval instant at 1010s sum_over_time(metric1[30s:10s] offset 10s)
-  {} 394
+  {} 297
 
 # Same result for different offsets due to step alignment.
 eval instant at 1010s sum_over_time(metric1[30s:10s] offset 9s)
```

Comment:
Interestingly, this test previously proved the shortcomings of the old definition: With the perfect alignment, you got a different result (394) from an imperfect alignment (297, see below). With the new definition, the result is always 297. So your change to the expected result in line 61 is fine, but the explanations around it needed to be updated.